main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/bitrise-tools/go-steputils/stepconf"
```

Comment:
I might be mistaken, but I think this is meant to point to bitrise-io/go-steputils.  On my test run, this package couldn't be found.

```suggestion
	"github.com/bitrise-io/go-steputils/stepconf"
```

Reply:
It does work, but you may have needed to run `go get github.com/bitrise-tools/go-steputils`?

Regardless, the bitrise-tools version is deprecated so it should be migrated to bitrise-io anyway. I have since fixed this in my fork, will add it to this PR.

Reply:
That makes sense - I've only added this step via the YAML editor and didn't add any additional go steps.  Given I'm running on the latest stable Xcode machine while trying to debug other issues with my test project it wouldn't surprise me if something changed around the deprecated step no longer being bundled.